### DISCUSSION TITLE
Welcome to the discussions for DSC v3

### DISCUSSION THREAD

**Discussion by theJasonHelmick** [PowerShell/PSDesiredStateConfiguration#75] [Announcements]
PSDesiredStateConfiguration (DSC) is the PowerShell module that enables writing configuration as code.

The DSC platform was originally built on top of WMI for Windows. Starting in PowerShell 7.1 and working with internal partner teams [Azure Guest Configuration](https://docs.microsoft.com/azure/governance/policy/concepts/guest-configuration) and [Automanage](https://azure.microsoft.com/services/azure-automanage), we started making DSC cross-platform by enabling Invoke-DSCResource to directly use resources without going through the Local Configuration Manager (LCM).

During the PowerShell 7.3 timeframe, we remain focused on enabling partner teams and will not be accepting public pull requests.

The documentation for PSDesiredStateConfiguration 3.0.0-beta1 is a work-in-progress. We invite the community to review the documentation and assist us as we work on new documentation during the platform development.
For more information about DSC v3, see [PowerShell Desired State Configuration Overview](https://docs.microsoft.com/powershell/dsc/overview?view=dsc-3.0)

As we continue this journey to make DSC a cross-platform technology, we invite the community to discuss and share your ideas here.

**Comment by claudio-salvio** [comment]
Hello @theJasonHelmick 
Ten months having passed since the announcement, I believe it is valid to ask: is DSC abandoned?
Those of us who believed in DSC, should we leave it behind and go in search of other alternatives?
Best regards,
Claudio Salvio

**Comment by theJasonHelmick** [comment]
@claudio-salvio See the Teams roadmap - Work on DSC v3 is progressing. https://devblogs.microsoft.com/powershell/powershell-openssh-team-investments-for-2023/

**Reply by OneCyrus** [reply]
the blog post states the following. Where can I find those RFCs? 

`Expect some detailed RFCs to be published on this within the first quarter of 2023 where you can provide feedback.`
